src/support/examples/deferred/shader/blit.glslf

```diff
@@ -1,4 +1,4 @@
-#version 150 core
+#version 140
```

Comment:
I don't think you can use GLSL_140 here since it has `out vec4 Target0`. You'll need to change the use into `gl_FragColor`